Google.PowerShell.Tests/Provider/GoogleCloudStorageProviderTests.cs

```diff
@@ -92,5 +102,27 @@ public void TestIssue580()
             Assert.IsNotNull(returnedBucket);
             Assert.AreEqual(bucketName, returnedBucket.Id);
         }
+
+        /// <summary>
+        /// Test the fix for an issue where the GcsProvider would hang.
+        /// </summary>
+        /// <seealso href="https://github.com/GoogleCloudPlatform/google-cloud-powershell/issues/594"/>
+        [Test]
+        [Timeout(200)]
+        public void TestIssue594()
```

Comment:
nit: I think we should have a descriptive name like `TestListProjectsThrownError` instead of `TestIssue594`.

Reply:
Yea, that was me being lazy. Fixed.